Week-4/exercise4.2.js

```diff
@@ -0,0 +1,17 @@
+let Person = function () { };
+Person.prototype.initialize = function (name, age)
+{
+    this.name = name;
+    this.age = age;
+}
+
+let Teacher = function () { };
+Teacher.prototype = new Person();
```

Comment:
you don't need to create an object, you can directly assign prototype like Person.prototype